NebulaClient/PacketProcessors/Factory/PasteEntitySettingUpdateProcessor.cs

```diff
@@ -0,0 +1,22 @@
+﻿using NebulaModel.Attributes;
+using NebulaModel.Networking;
+using NebulaModel.Packets.Factory;
+using NebulaModel.Packets.Processors;
+using NebulaWorld.Factory;
+
+namespace NebulaClient.PacketProcessors.Factory
+{
+    [RegisterPacketProcessor]
+    class PasteEntitySettingUpdateProcessor : IPacketProcessor<PasteEntitySettingUpdate>
+    {
+        public void ProcessPacket(PasteEntitySettingUpdate packet, NebulaConnection conn)
+        {
+            EntitySettingDesc backup = EntitySettingDesc.clipboard;
+            EntitySettingDesc.clipboard = packet.GetEntitySettings();
+            FactoryManager.EventFromServer = true;
+            GameMain.localPlanet.factory.PasteEntitySetting(packet.EntityId);
```

Comment:
I think this is wrong, you should get the factoryId or planetId from the packet. Else you will update your local planet entity with the entity data of another planet

Reply:
right, I will fix it :)